pkg/controllers/scheduler/framework/plugins/clusterresources/most_allocated_test.go

```diff
@@ -75,6 +75,28 @@ func TestClusterResourcesMostAllocated(t *testing.T) {
 			},
 			name: "nothing scheduled, resources requested, differently sized machines",
 		},
+		{
+			// Cluster1 scores on 0-100 scale
+			// CPU Fraction: 3000 / 4000= 75%
+			// Memory Fraction: 5000 / 10000 = 50%
+			// GPU Fraction: 5000 / 10000 = 50%
```

Comment:
Could we add another test case where the GPU fractions differ (to show the effect of the 1:1:4 ratio).

Reply:
Added.